src/components/ButtonPanel.js

```diff
@@ -1,40 +1,12 @@
-import React from 'react';
+import PropTypes from 'prop-types';
 
-import Button from './Button';
+export default function ButtonPanel({ children }) {
+  return <div className="button_panel">{children}</div>;
+}
```

Comment:
- [ ] This component should implement the `handleClick` function. 
- [ ] It should then pass the `handleClick` as `clickHandler` props to each `Button`.